contracts/Core/StakeManager.sol

```diff
@@ -147,6 +147,7 @@ contract StakeManager is Initializable, StakeStorage, StateManager, Pause, Stake
         require(stakers[stakerId].acceptDelegation, "Delegetion not accpected");
         require(_isStakerActive(stakerId, epoch), "Staker is inactive");
         require(!stakers[stakerId].isSlashed, "Staker is slashed");
+        require(stakers[stakerId].stake >= minStake, "Staker stake less than minStake");
```

Comment:
Hmm whats the purpose of doing this ?

Reply:
Hmm got it

Reply:
Do we really need this but
*Extra Check per Delegation*
This is going to be checked for each delegation 
And as a delegator I should do my own research before delegating 

Though following is rare
If stake is below minStake I can sponser staker to run
With this PR I cant do that